bundles/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/bus/KNXBinding.java

```diff
@@ -108,7 +108,7 @@ public void removeKNXTypeMapper(KNXTypeMapper typeMapper) {
     @Override
     protected void internalReceiveCommand(String itemName, Command command) {
         logger.trace("Received command (item='{}', command='{}')", itemName, command.toString());
-        if (!isEcho(itemName, command)) {
+        if (!KNXConnection.getCompatibilityOH2() && !isEcho(itemName, command)) {
```

Comment:
@watou, 
I think about a simple cache, loosing its record data after a short period of time.... But I still have to ponder. But I believe it is superfluous, because in no test I've done, I could throw the ignoreEventList out of the rhythm.

One thing I can say, the [autoupdate](https://github.com/openhab/openhab/blob/master/bundles/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/config/KNXGenericBindingProvider.java#L347) function in KNX 1.8 binding is never ever called, running in OH1 nor OH2.
Why? Is this behavior expected?
Is not called in OH1 and OH2, therefore I think this is not the cause for this difference behavior between OH1/OH2.

This [PR](https://github.com/openhab/openhab/pull/4179) cannot be the cause, it is only some months old. I know about this problem and the differences more about a year.
But I will test this!!